libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -139,27 +142,43 @@ export class MessageRepository extends BaseRepository<EnforceEnvironmentQuery, M
   }
 
   async updateFeedByMessageTemplateId(environmentId: string, messageId: string, feedId: string) {
-    return this.update(
-      { _environmentId: environmentId, _messageTemplateId: messageId },
-      {
-        $set: {
-          _feedId: feedId,
-        },
-      }
-    );
-  }
+    await this.cleanUselessKeysFromStore(environmentId, messageId);
```

Comment:
So here the invalidation is not necessary and it is basically in order to clean the Redis because after this request the query for the messages will be different therefore there will be a new key for them.
If we decide to keep it I would love to get some help in order to refactor the function because I am. sure we could merge the query and the reduction in it. (unfortinutly i did no manage to do so 😅)

Reply:
Decided to remove it.